test/org/daisy/dotify/formatter/impl/obfl/ObflWsXsltTest.java

```diff
@@ -149,6 +149,9 @@ public int testWsNormalizer(String input, String expected, String id, boolean ke
 		if (!keep && !in.delete()) {
 			in.deleteOnExit();
 		}
+		if (keep && normalizedFile.isFile()) {
+			System.out.println(normalizedFile.getAbsolutePath());
```

Comment:
Please remove these lines

Reply:
Why? I did it on purpose.

Reply:
I think it is neater to use something like `logger.log(Level.INFO, ...)`

Reply:
It doesn't really matter does it? It's just tests. I did the same thing as was already done [here](https://github.com/mtmse/dotify.formatter.impl/blob/fb2af7f936322abcf5fa77fd13dfa617d3650931/integrationtest/org/daisy/dotify/formatter/test/AbstractFormatterEngineTest.java#L58).